### PR TITLE
Update images for release 0.8.0-rc1

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- Update images used in release

**Related issue(s)**

### CODE REVIEW

File: resources/ark/values.yaml [hunk 1]
```diff
@@ -71,5 +71,5 @@ global:
   containerRegistry:
```
Comment (aszecowka) [inline resources/ark/values.yaml]: here I think we should also bump eu.gcr.io/kyma-project/etcd-tls-setup

File: resources/event-bus/values.yaml [hunk 2]
```diff
@@ -5,11 +5,11 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 9e7cfa58
+    dir:
```
Comment (montaro) [inline resources/event-bus/values.yaml]: @suleymanakbas91 What should this `event-bus` point to?
we don't have images in the root directory, no?

Comment (sayanh) [reply resources/event-bus/values.yaml]: @montaro For release builds the images are not in `develop/`. Please refer https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/3099/pre-rel08-kyma-components-event-bus/1105128449070075904

Comment (suleymanakbas91) [reply resources/event-bus/values.yaml]: @montaro here is how we use these values inside a chart: https://github.com/kyma-project/kyma/blob/master/resources/event-bus/charts/publish-knative/templates/deployment.yaml#L29. Of course, we have images in the root directory: https://console.cloud.google.com/gcr/images/kyma-project/EU/event-bus-publish-knative

File: installation/resources/watch-pods.yaml [hunk 3]
```diff
@@ -48,7 +48,7 @@ metadata:
 spec:
   containers:
   - name: watch-pods
-    image: eu.gcr.io/kyma-project/develop/watch-pods:c3c08515
```
Comment (suleymanakbas91) [inline installation/resources/watch-pods.yaml]: `watch-pod` has been recently moved to test-infra. So, it doesn't have `0.8.0-rc1` tag. Either revert this or release it from test-infra.

Comment (sayanh) [reply installation/resources/watch-pods.yaml]: done

File: resources/ark/values.yaml (hunk 1 above)
Comment (sayanh) [reply resources/ark/values.yaml]: done

File: resources/event-bus/values.yaml (hunk 2 above)
Comment (montaro) [reply resources/event-bus/values.yaml]: @suleymanakbas91 this is what I really mean 
https://console.cloud.google.com/gcr/images/kyma-project/EU/event-bus?gcrImageListsize=30

**Pull request merged**